### PR TITLE
Add content to BioCro model description

### PR DESCRIPTION
**Pull request by KristinaRiemer**
There's no information in section 27 of the book about specifics for running BioCro. 

## Description
Added some content to the template. 

## Motivation and Context
The more information about each of the models, the more easily users can run them within PEcAn. There's an issue for this in a different repo [here](https://github.com/danforthcenter/sentinel-detection/issues/24). 

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
I've pinged David as a reviewer for this, but whoever is familiar with BioCro (and probably knows more about the model than I do) should feel free to look at this.

File: book_source/06_reference/02_models/biocro.Rmd [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
+  - `inputs`
+    - `met/output` -- Set as BIOCRO
+    - `met/path` -- Path to file containing meteorological data
+
+### Model specific input files
 
 List of inputs required by model, such as met, etc.
 
-**Model configuration files**
+### Model configuration files
```
Comment (infotroph) [inline book_source/06_reference/02_models/biocro.Rmd]: Would this be the correct place to (eventually, not necessarily in this PR) document the genus-specific parameter files? They're secretly required, just usually looked up from the package. 

My understanding is that `write.configs.BIOCRO` looks for defaults in this order: first any file at a path specified by `settings$pft$constants$file`, next by matching the genus name in datasets exported by the `BioCro` package, last by matching the genus name in `PEcAn.BIOCRO`'s `extdata/defaults` directory.

File: book_source/06_reference/02_models/biocro.Rmd [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
 
-**Model specific input files**
+- `model`
+  - `revision` -- Model version number
+- `run`
+  - `site/id` -- ID associated with desired site from BETYdb site entry
+  - `inputs`
+    - `met/output` -- Set as BIOCRO
+    - `met/path` -- Path to file containing meteorological data
+
```
Comment (infotroph) [inline book_source/06_reference/02_models/biocro.Rmd]: 👍

File: book_source/06_reference/02_models/biocro.Rmd [hunk 3]
```diff
@@ -1,31 +1,39 @@
...
-| Model Information ||
-| -- | -- |
-| Home Page | |
-| Source Code | |
-| License |  |
-| Authors | |
-| PEcAn Integration | |
+| Model Information |                                                                   |
+| --                | --                                                                |
+| Home Page         | http://biocro.r-forge.r-project.org/                              |
```
[the remark this answers is not in the document]
Comment (KristinaRiemer) [reply book_source/06_reference/02_models/biocro.Rmd]: I couldn't find any other possible home pages, seems like all the recent stuff is in the GH repo.

File: book_source/06_reference/02_models/biocro.Rmd (hunk 1 above)
Comment (KristinaRiemer) [reply book_source/06_reference/02_models/biocro.Rmd]: This seems like good information to add to me! I didn't know anything about this, though having this work under the hood is some of why Pecan is useful.

**Comment by KristinaRiemer** [comment]
@dlebauer does this look okay to merge?

File: book_source/06_reference/02_models/biocro.Rmd [hunk 4]
```diff
@@ -1,31 +1,39 @@
...
 
 List of inputs required by model, such as met, etc.
 
-**Model configuration files**
+### Model configuration files
 
 BioCro uses a config.xml file similar to ED2. At this time, no other template files are required.
 
-**Installation notes**
+### Installation notes
```
Comment (dlebauer) [inline book_source/06_reference/02_models/biocro.Rmd]: add `install_github('ebimodeling/biocro')` and other info from the readme to here?

File: book_source/06_reference/02_models/biocro.Rmd (hunk 3 above)
Comment (dlebauer) [inline book_source/06_reference/02_models/biocro.Rmd]: r-forge is to an earlier version I think - a better home page might be just linking to the README in the github biocro repo.
```suggestion
| Home Page         | https://github.com/ebimodeling/biocro/blob/master/README.md |
```

File: book_source/06_reference/02_models/biocro.Rmd [hunk 5]
```diff
@@ -1,31 +1,39 @@
...
-| Source Code | |
-| License |  |
-| Authors | |
-| PEcAn Integration | |
+| Model Information |                                                                   |
+| --                | --                                                                |
+| Home Page         | http://biocro.r-forge.r-project.org/                              |
+| Source Code       | https://github.com/ebimodeling/biocro                             |
+| License           |                                                                   |
+| Authors           | Fernando E. Miguez, Deepak Jaiswal, Justin McGrath, David LeBauer |
```
Comment (dlebauer) [inline book_source/06_reference/02_models/biocro.Rmd]:
```suggestion
| Authors           | Fernando E. Miguez, Deepak Jaiswal, Justin McGrath, David LeBauer, Scott Rohde, Dan Wang |
```

File: book_source/06_reference/02_models/biocro.Rmd (hunk 1 above)
Comment (dlebauer) [inline book_source/06_reference/02_models/biocro.Rmd]:
```suggestion
### Model configuration files

Genus-specific parameter files are secretly required, and are stored in the PEcAn.BIOCRO package and looked up under the hood. 

`write.configs.BIOCRO` looks for defaults in this order: first any file at a path specified by `settings$pft$constants$file`, next by matching the genus name in datasets exported by the BioCro package, last by matching the genus name in PEcAn.BIOCRO's extdata/defaults directory.

When adding a new genus, it is necessary to provide a new default parameter file in PEcAn.BIOCRO [`inst/extdata/defaults`](https://github.com/PecanProject/pecan/tree/develop/models/biocro/inst/extdata/defaults) and also (for v<1.0) update the `run.biocro()` function.
```

File: book_source/06_reference/02_models/biocro.Rmd [hunk 6]
```diff
@@ -1,31 +1,39 @@
...
-| Home Page | |
-| Source Code | |
-| License |  |
-| Authors | |
-| PEcAn Integration | |
+| Model Information |                                                                   |
+| --                | --                                                                |
+| Home Page         | http://biocro.r-forge.r-project.org/                              |
+| Source Code       | https://github.com/ebimodeling/biocro                             |
+| License           |                                                                   |
```
Comment (dlebauer) [inline book_source/06_reference/02_models/biocro.Rmd]:
```suggestion
| License           |   [University of Illinois/NCSA Open Source License](https://github.com/ebimodeling/biocro/blob/master/LICENSE)                                                                |
```

File: book_source/06_reference/02_models/biocro.Rmd (hunk 1 above)
Comment (dlebauer) [reply book_source/06_reference/02_models/biocro.Rmd]: also need to edit comment below 'no other template files are required'

Comment (KristinaRiemer) [reply book_source/06_reference/02_models/biocro.Rmd]: How do you want the line edited?

**Comment by KristinaRiemer** [comment]
@dlebauer I've incorporated all your suggestions except I'm not sure how you wanted line 39 to change?

File: book_source/06_reference/02_models/biocro.Rmd [hunk 7]
```diff
@@ -1,31 +1,43 @@
...
 
-BioCro uses a config.xml file similar to ED2. At this time, no other template files are required.
+Genus-specific parameter files are secretly required. These are stored in the PEcAn.BIOCRO package and looked up under the hood. 
+
+`write.configs.BIOCRO` looks for defaults in this order: first any file at a path specified by `settings$pft$constants$file`, next by matching the genus name in datasets exported by the BioCro package, and last by matching the genus name in PEcAn.BIOCRO's extdata/defaults directory.
 
-**Installation notes**
+When adding a new genus, it is necessary to provide a new default parameter file in PEcAn.BIOCRO [`inst/extdata/defaults`](https://github.com/PecanProject/pecan/tree/develop/models/biocro/inst/extdata/defaults) and also (for v<1.0) update the `call_biocro()` function.
+
+BioCro uses a config.xml file similar to ED2. At this time, no other template files are required.
```
Comment (dlebauer) [inline book_source/06_reference/02_models/biocro.Rmd]:
```suggestion
```

**Comment by dlebauer** [comment]
merged just now

**Pull request merged**